frontend/app/dashboard/layout.js

```diff
@@ -0,0 +1,8 @@
+export default function DashboardLayout({ children }) {
```

Comment:
Do you have to have a layout component? This one isn't actually doing anything right, so if you delete it will Next give you a problem?